src/Compilers/CSharp/Portable/Syntax/InternalSyntax/CSharpSyntaxNode.cs

```diff
@@ -53,11 +53,6 @@ internal CSharpSyntaxNode(SyntaxKind kind, DiagnosticInfo[] diagnostics, SyntaxA
             GreenStats.NoteGreen(this);
         }
 
-        internal CSharpSyntaxNode(ObjectReader reader)
```

Comment:
a lot of the core syntax types that follow will just see their internal serialization support removed.